### DISCUSSION TITLE
Authors page

### DISCUSSION THREAD

**Discussion by engeir** [retypeapp/retype#544] [Q&A] [answered]
Is it possible to generate an "authors" page, similar to what is done for "categories" and "tags", i.e., `https://retype.com/authors/`? Maybe with the same styling as those two, or where the avatar is shown with the name under in a grid structure?

**Comment by geoffreymcgill** [comment, marked as answer]
Excellent idea. Love it. 

At the moment, Retype cannot generate this automatically, but it is certainly something we can try and build into a future release.

**Reply by geoffreymcgill** [reply]
We are tracking this idea as a feature request. 

I can't say when (or if) this feature will be implemented, but it's a great idea, so I'd like to make it happen.
